source/features/fix-no-pr-search.tsx

```diff
@@ -28,6 +28,7 @@ function init(signal: AbortSignal): void {
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.isRepoPRList,
```

Comment:
I think this can be dropped

Can you also add a test URL at the bottom?

Reply:
👌